src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int ballNumber = random.nextInt(100); // Генерує випадковий номер кулі
```

Comment:
Do not add non-English comments

![image](https://github.com/mate-academy/jv-lottery/assets/20326858/7b5ea529-1a1d-4a67-b30d-52c462213473)